crates/orderbook/src/dto/order.rs

```diff
@@ -68,3 +68,46 @@ pub enum FeePolicy {
         quote: Quote,
     },
 }
+
+#[serde_as]
+#[derive(Serialize, PartialEq, Debug, Clone)]
+#[cfg_attr(any(test, feature = "e2e"), derive(serde::Deserialize))]
+#[serde(rename_all = "camelCase")]
+pub struct ExecutedAmounts {
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub sell: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub buy: U256,
+}
+
+#[derive(Serialize, PartialEq, Debug, Clone)]
+#[cfg_attr(any(test, feature = "e2e"), derive(serde::Deserialize))]
+#[serde(rename_all = "camelCase")]
+pub struct SolutionInclusion {
```

Comment:
I feel like this struct should get some documentation (the name is still not great so explaining what this means might be helpful).